acceptance/indy_acceptance/indy-tests/test_scripts/acceptance_tests/verify_messages_on_connection.py

```diff
@@ -0,0 +1,59 @@
+"""
+Created on Nov 8, 2017
+
+@author: khoi.ngo
+
+Containing test script of test scenario 02: verify messages on connection.
+"""
+
+import json
+import os
+from indy import pool
+from libraries.constant import Constant, Colors
+from libraries.result import Status
+from libraries.utils import perform
+from test_scripts.test_scenario_base import TestScenarioBase
+
+""" cmds """
+back_up_pool_genesis_file = 'sudo cp ' + Constant.pool_genesis_txn_file + " " + Constant.original_pool_genesis_txn_file
+remove_pool_genesis_file = 'sudo rm ' + Constant.pool_genesis_txn_file
+restore_pool_genesis_file = 'sudo cp ' + Constant.original_pool_genesis_txn_file + " " + Constant.pool_genesis_txn_file
+create_empty_pool_genesis_file = 'sudo touch ' + Constant.pool_genesis_txn_file
+
+
+class TestScenario02(TestScenarioBase):
```

Comment:
It is a good practice to name a module correspondingly (except for the case) to the name of the class contained in it (at least in case the module contains a sole class only). Consider renaming test scenario modules or classes for they to correspond with each other.

Reply:
Updated. Please refer to the latest code.